### PR TITLE
In-Patient button added

### PR DESCRIPTION
**Pull request by AzukaUgbekile**
In-patient button

### CODE REVIEW

**Review by BorisLaboratory — CHANGES_REQUESTED** [review]
Editing nav links seems out of scope

File: src/components/MenuItem.jsx [hunk 1]
```diff
@@ -27,19 +27,19 @@ function MenuItem() {
...
           </ListItem>
           <ListItem>
             <Link
               component={RouterLink}
-              to="/employees"
+              to="/patients"
               sx={{ textDecoration: "none" }}
             >
-              Employee Management
+              InPatient Management
```
Comment (BorisLaboratory) [inline src/components/MenuItem.jsx:42]: this functionality on nav links seems out of scope

File: src/pages/Admin.jsx [hunk 2]
```diff
@@ -8,8 +8,8 @@ const Home = () => {
         display: "flex",
         flexWrap: "wrap",
         gap: 6,
-        height: "70vh",
-        width: "70vw",
+        height: "100vh",
```
Comment (BorisLaboratory) [inline src/pages/Admin.jsx:11]: not sure if this change was necessary by actual viewing of the buttons original positions on the screen

File: src/pages/Admin.jsx [hunk 3]
```diff
@@ -8,8 +8,8 @@ const Home = () => {
...
+        width: "100vw",
```
Comment (BorisLaboratory) [inline src/pages/Admin.jsx:12]: not sure if this change was necessary by actual viewing of the buttons original positions on the screen

**Review by BorisLaboratory — COMMENTED** [review]
Looks good ...just few observations

File: src/components/MenuItem.jsx (hunk 1 above)
Comment (AzukaUgbekile) [reply src/components/MenuItem.jsx:42]: Please i don't understand

File: src/pages/Admin.jsx (hunk 2 above)
Comment (AzukaUgbekile) [reply src/pages/Admin.jsx:11]: If these changes were not made, the 3 buttons will not be on a straight line

File: src/components/MenuItem.jsx (hunk 1 above)
Comment (BorisLaboratory) [reply src/components/MenuItem.jsx:42]: Oh if you look at the user story critically it's asking for button functionality. There is a key phrase in the criteria that says "all three buttons" .. not links. 
... I'm just wondering why the MenuItem showed up in this PR. Please advice why it showed up in the PR

**Comment by AzukaUgbekile** [comment]
Oh......that was a mistake. I will make the necessary correction  before or
after the meeting. Just wondering, if the code is okay, can't this be
rectified from the github?
